lib/src/lib/button/directives/preventDoubleClick.directive.ts

```diff
@@ -0,0 +1,36 @@
+import { Directive, EventEmitter, HostListener, Output } from '@angular/core';
+import { Subject, timer } from 'rxjs';
+import { throttle } from 'rxjs/operators';
+
+@Directive({
+  selector: '[lab900PreventDoubleClick]',
+  standalone: true,
+})
+export class PreventDoubleClickDirective {
+  private clicks = new Subject<Event>();
+  private isThrottled = false;
+
+  @Output() public throttledClick = new EventEmitter<Event>();
```

Comment:
I'm wondering, if we rename this to `click`, can we override the default `(click)=...` output of buttons? that might be nice